src/measure_theory/vector_measure.lean

```diff
@@ -427,4 +427,94 @@ end
 
 end measure
 
+namespace vector_measure
+
+section
+
+variables [measurable_space β]
+variables {M : Type*} [add_comm_monoid M] [topological_space M]
+
+/-- The pullback of a vector measure along a measurable function. -/
+def map (v : vector_measure α M) (f : α → β) (hf : measurable f) :
```

Comment:
I'd drop the `measurable f` assumption here, and define `map v f` to be zero if `f` is not measurable, just like it's done for nonnegative measures (see Line 582 of `measure_space.lean`). Also, can you directly register this as an additive group morphism?

Reply:
This definition doesn't require `has_continuous_add M` while `add` on vector measures requires it. I will make it a group hom after defining it without a hom structure first. Is that a good idea?

Reply:
I've added the linear map version as well though I'm not sure how useful it is